first_hour_on_terra/batch_template.csv

```diff
@@ -0,0 +1,4 @@
+TERRA_USER_EMAIL,POLICY,
+amyu@google.com,MEMBER,
```

Comment:
Lets put example emails here instead of actual emails. EXAMPLE@DOMAIN.com is fine, for example.

Reply:
thanks for catching -- fixed!